lua/bufferline/tabpages.lua

```diff
@@ -7,6 +7,8 @@ local pick = lazy.require("bufferline.pick")
 local config = lazy.require("bufferline.config")
 ---@module "bufferline.constants"
 local constants = lazy.require("bufferline.constants")
+--- @module "bufferline.duplicates"
+local duplicates = require("bufferline.duplicates")
```

Comment:
```suggestion
local duplicates = lazy.require("bufferline.duplicates")
```

`lazy.require` lazy loads lua modules to defer the cost of requiring so many modules till the first time that they are accessed, e.g. `module.field` rather than when this file is first read. That's the reason why I annotate them as `---@module` this allows sumneko to know what the underlying lazy loaded module is